flake.nix

```diff
@@ -81,7 +81,7 @@
         cheriot = pkgs.mkShell {
```

Comment:
Re. b7bb91740, the commit message doesn't seem to match the changes? The change is just snake -> kebab